cmd/glasskube/cmd/bootstrap.go

```diff
@@ -31,6 +33,33 @@ var bootstrapCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		cfg, _ := cliutils.RequireConfig(config.Kubeconfig)
 		client := bootstrap.NewBootstrapClient(cfg)
+
+		installedVersion, err := clientutils.GetPackageOperatorVersion(cmd.Context())
+		if err != nil {
+			IsBootstrapped, err := bootstrap.IsBootstrapped(cmd.Context(), cfg)
+			if err != nil && !IsBootstrapped {
+				fmt.Printf("error : %v\n", err)
+				cliutils.ExitWithError()
+			}
+		}
+
+		var desiredVersion string
+		if bootstrapCmdOptions.url == "" {
+			desiredVersion = config.Version
+		} else {
+			desiredVersion = ""
+		}
+		if !semver.IsUpgradable(installedVersion, desiredVersion) && installedVersion != "" {
```

Comment:
Sorry, I realized one more thing: If `installedVersion` and `desiredVersion` are the same, the prompt also appears, this should not happen. Because it's fine to bootstrap v0.3.0 again in a cluster where v0.3.0 is already running. 

So I think this should be fixed in this `if` by adding `&& installedVersion[1:] != desiredVersion`.